src/D2L.CodeStyle.TestAnalyzers/NUnit/CategoryAnalyzer.cs

```diff
@@ -189,11 +197,16 @@ ISymbol symbol
 
 			TypedConstant arg = args[0];
 
-			if( arg.Type.SpecialType != SpecialType.System_String ) {
+			if( arg.Type?.SpecialType != SpecialType.System_String ) {
+				return;
+			}
+
+			string? category = arg.Value as string;
+
+			if( category == null ) {
 				return;
 			}

```

Comment:
If Roslyn says it's a string, but it's somehow not actually a string, I feel like that's worth an exception.